core/frontend/src/components/wizard/ActionStepper.vue

```diff
@@ -49,6 +59,9 @@ export default Vue.extend({
       return 'mdi-alert'
     },
     getStepColor(config: Configuration) {
+      if (config.skip) {
+        return 'black'
```

Comment:
This looks a bit odd... Can we add an 'X' or a '/' over the icon maybe?

Reply:
maybe gray for a neutral feel?

Reply:
> maybe gray for a neutral feel?

I did try gray, but the contrast was terrible on white mode. black was the one that worked fine for white and dark mode.